src/main/java/com/gmail/tracebachi/FloorIsLava/Arena.java

```diff
@@ -307,13 +311,7 @@ public void tick()
 
                 Firework firework = player.getWorld().spawn(player.getLocation().add(0, 1, 0), Firework.class);
                 FireworkMeta fireworkMeta = firework.getFireworkMeta();
-                fireworkMeta.addEffects(FireworkEffect.builder()
-                    .flicker(false)
-                    .trail(true)
-                    .with(Type.BALL_LARGE)
-                    .withColor(Color.BLUE)
-                    .withFade(Color.WHITE)
-                    .build());
+                fireworkMeta.addEffects(FireworkEffect.builder().flicker(false).trail(true).with(Type.BALL_LARGE).withColor(Color.BLUE).withFade(Color.WHITE).build());
```

Comment:
@Fridgey Avoid long lines like this. Use the format I had before. It's easier to read.